strawberryfields/apps/qchem/dynamics.py

```diff
@@ -92,51 +92,40 @@
 from strawberryfields.utils import operation
 
 
-def evolution(modes: int):
-    r"""Generates a custom ``sf`` operation for performing the transformation
-    :math:`U(t) = U_l e^{-i\hat{H}t/\hbar} U_l^\dagger` on a given state.
+def TimeEvolution(w: np.ndarray, t: float):
+    r"""An operation for performing the transformation
+    :math:`e^{-i\hat{H}t/\hbar}` on a given state.
```

Comment:
Is this statement general?. Can you use this function to simulate the the real-time dynamics for any Hamiltonian or is it constrained to the case of independent quantum harmonic oscillators?.

Reply:
This is a great point! This applies only to evolution under the free Hamiltonian of independent quantum harmonic oscillators. So we can add something like "where :math:`H = \sum_k \hbar \omega_w (\hat{n_k} + 1/2)` is the free Hamiltonian of the system"